src/wmnds/patterns/footer/_footer.njk

```diff
@@ -21,16 +22,32 @@
         <a href="#" class="wmnds-btn wmnds-footer__btn wmnds-footer__btn--block">Sign up to updates</a>
       </div>
       {# loop through footer menus and get items #}
-      {% for menu in params.menus -%}
+      {% if(params.menu1) %}
         <div class="wmnds-col-1 wmnds-col-lg-1-3">
-          <h3 class="wmnds-footer__heading wmnds-collapse-heading" aria-controls="footer-menu-{{loop.index}}" aria-expanded="true">{{menu.name}}<svg class="wmnds-collapse-heading__indicator"><use xlink:href="#wmnds-general-chevron-right" href="#wmnds-general-chevron-right"></use></svg></h3>
-          <ul class="wmnds-footer__menu wmnds-collapse-panel" id="footer-menu-{{loop.index}}">
-          {% for menuItem in menu.menuItems-%}
+          <h3 class="wmnds-footer__heading wmnds-collapse-heading" aria-controls="footer-menu-{{params.menu1.name}}" aria-expanded="true">{{params.menu1.name}}<svg class="wmnds-collapse-heading__indicator"><use xlink:href="#wmnds-general-chevron-right" href="#wmnds-general-chevron-right"></use></svg></h3>
+          <ul class="wmnds-footer__menu wmnds-collapse-panel" id="footer-menu-{{params.menu1.name}}">
+          {% for menuItem in params.menu1.menuItems-%}
```

Comment:
Using `aria-controls="footer-menu-{{params.menu1.name}}"` means that you could end up with `aria-controls="footer-menu-Resources & support"`

You could use the nunjucks filters to make the titles into hyphenated strings. e.g. `{{params.menu1.name | replace(r/[^\w\ ]/gi, '') | replace(' ', '-') | lower}}` would give `aria-controls="footer-menu-resources-support"` for the example above.

It's a bit more obtuse but it would mean that you wouldn't have to define `menu1` and `menu2` in params, and use an array instead.

Reply:
Good shout.

Reply:
Ah okay, yeah that's fine 👍🏾